terraform/modules/hub/hub_frontend.tf

```
@@ -109,6 +109,36 @@ resource "aws_ecs_service" "frontend" {
   }
 }
 
+resource "aws_ecs_service" "frontend_v2" {
+  name            = "${var.deployment}-frontend"
```

```suggestion
  name            = "${var.deployment}-frontend-v2"
```

I do not believe you can have two services with the same name